docs/docs/self-hosting/manage/configure/configure.mdx

```diff
@@ -10,40 +10,45 @@ import Helm from './_helm.mdx'
 
 # Configure ZITADEL
 
-This guide assumes you are already familiar with [running ZITADEL with the most minimal configuration possible](/docs/self-hosting/deploy/overview).
+This guide assumes you are familiar with [running ZITADEL using the least amount of configuration possible](/docs/self-hosting/deploy/overview).
 
 ## Configuration Files
 
-### Runtime Configuration
-See a description of all possible _runtime configuration_ options with their defaults [in the source code](https://github.com/zitadel/zitadel/blob/main/cmd/defaults.yaml).
-The `zitadel` binary expects the `--config` flag for this configuration.
+### Runtime Configuration File
 
-### Database Initialization
-Apart from these options, ZITADEL uses a [different configuration](https://github.com/zitadel/zitadel/blob/main/cmd/setup/steps.yaml) for _database initialization steps_.
-The `zitadel` binary expects the `--steps` flag for this configuration.
+You can configure the runtime using the `--config` flag of the `zitadel` binary.
+For a description of all _runtime configuration options_ and their defaults, read the [ZITADEL source code](https://github.com/zitadel/zitadel/blob/main/cmd/defaults.yaml).
 
-### Split Configuration
-ZITADEL merges multiple configurations passed via multiple `--config` and/or multiple `--steps` flags.
-This is especially useful, if you want to treat secret configuration files differently from standard configuration.
-For example, you can check configuration with information like the database hostname in to git.
-Other configration that contains information like your databases admin username and password, you can hand to a secret manager.
+### Database Initialization File
 
-## Environment variables
+ZITADEL uses a [different configuration file](https://github.com/zitadel/zitadel/blob/main/cmd/setup/steps.yaml) for _database initialization steps_.
+Use the `--steps` flag of the `zitadel` binary to provide this configuration file.
 
-All configuration properties are also configurable via environemnt variables.
-Prefix the key with *ZITADEL\_*, uppercase the propery and join sections by an underscore _.
-For example, if you want to configure the default ZITADEL IAM admin username and password, make sure the ZITADEL binary runtime has the variables *ZITADEL_FIRSTINSTANCE_ORG_HUMAN_USERNAME* and *ZITADEL_FIRSTINSTANCE_ORG_HUMAN_PASSWORD* set.
+### Multiple Configuration Files
 
-### Proxy configuration
+ZITADEL merges configuration files when multiple `--config` and `--steps` flags are provided.
+You can use these flags to treat secret configuration files differently from standard configuration files.
+For example, configuration files stored in git may contain public information such as a database hostname.
+To avoid storing private information — such as a database admin credential — in git, use a `--steps` flag that requests the private information from a secret manager.
```

Comment:
```suggestion
To avoid storing private information — such as a database admin credential — in git, use an extra `--config` or `--steps` flag that references a file with private information that is provided by a secret manager.
```

Reply:
Thanks, I will send an email later today.

I implemented the first part of this suggestion to clarify that you can use the config flag as well.

In my opinion, "references a file with private information that is provided by a" is wordy and lacks context that the file (containing private information) should be requested from an external source. It may be a hidden file on the machine, but it could also be an external secret manager (i.e Nomad Vault). With that being said, the decision to approve this edit is up to you since you are a maintainer of the repository.